pkgs/development/tools/okteto/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "okteteo";
+  version = "1.8.15";
+
+  goPackagePath = "github.com/okteto/okteto";
```

Comment:
This needs to be removed but I am not sure why ofborg does not fail because it does when I manually build it.

```
error: `goPackagePath` is not needed with `buildGoModule`
```